malware-profiles/YYYY_NameOfMalware.md

```diff
@@ -1,37 +0,0 @@
-# :recycle:Name of Malware
-
-_Malware profile by **:recycle:Name Student 1** and **:recycle:Name Student 2**_
-
-### Classification
-
-_:bulb: Remove all :heavy_check_mark: which do not apply from the table below!_
-
-| Virus              | Worm               | Trojan             | Ransomware         | Botnet             | Other                                   |
-|:-------------------|:-------------------|:-------------------|:-------------------|:-------------------|:----------------------------------------|
-| :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: (:recycle:Type name) |
-
-### Facts & Figures
-
-* **Year:** _:bulb: Year of first occurence or discovery in `YYYY` format. Make sure you refer to your source via footnote!_
-* **Author:** _:bulb: The malware author(s) or "unknown" if not known. Make sure you refer to your source via footnote!_
-* **Language:** _:bulb: The programming language the malware is programmed in. Make sure you refer to your source via footnote!_
-* **Infections:** _:bulb: The number of documented or estimated infections or "unknown" if not known. Make sure you refer to your source via footnote!_
-* **Damage:** _:bulb: The total economic damage (if any) of the malware or "unknown" if not known. Make sure you refer to your source via footnote!_
-
-### Description
-
-_:bulb: Provide a brief but self-sufficient summary about the malware. If you quote external sources, use citation (`>`) and refer to the origin with a footnote!_
-
-_:bulb: How footnotes work in Markdown documents:_
-
-```
-> Here is some quoted text.[^1]
-
-[^1]: http://source-of-our-quote.blubb
-```
-
-_:bulb: The total length of the malware profile should be a single DIN A4 page (vertical) when printed!_
-
-_:bulb: There should be no :recycle: icons and :bulb: sections left in the final profile!_
-
-_:bulb: You can find a complete example profile in [`1999_Melissa.md`](1999_Melissa.md)._
```

Comment:
Please don't remove the template

Reply:
Sorry, i forgot to copy the template before editing it. Now it's back.